sdks/go/pkg/beam/runners/dataflow/dataflow.go

```diff
@@ -81,6 +81,7 @@ func init() {
 	beam.RegisterRunner("DataflowRunner", Execute)
 
 	perf.RegisterProfCaptureHook("gcs_profile_writer", gcsRecorderHook)
+	pipelinex.IdempotentNormalize = true
```

Comment:
I think this is flipping this flag for multiple runners if the beamx package is used, since it imports multiple runners. Although I don't think that's actually an issue, but if we're going to activate it for all runners, let's do it explicitly by setting this to true by default in the original declaration.

Or if we only want Dataflow using this, let's put this line somewhere in the Dataflow runner's execution path so it only gets called if the Dataflow runner is being used.

Reply:
Switched to enabling for everything by default. I'll need to be adding the flag to toggle it on Go SDK import. Probably simpler anyway.